### PR TITLE
Io pattern change for busybox-app1

### PR DESCRIPTION
**Pull request by shylesh**
1. Change IO pattern with random ios and random size of files
2. Pods will do wget of the script instead of inline code in deployment yaml

### CODE REVIEW

**Review by sidhant-agrawal — APPROVED** [review]

File: rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml [hunk 1]
```diff
@@ -6,9 +6,11 @@ metadata:
   labels:
     app: busybox-sample
 spec:
-  preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  #preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  preferredCluster: "sagrawal-c1"
   drPolicyRef:
-    name: PLACEHOLDER-DRPOLICY-NAME
+    #name: PLACEHOLDER-DRPOLICY-NAME
+    name: odr-policy-5m
```
Comment (sidhant-agrawal) [inline rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml:13]: Needs to be reverted.

File: rdr/busybox/app-busybox-1/subscriptions/busybox/subscription.yaml [hunk 2]
```diff
@@ -3,7 +3,8 @@ apiVersion: apps.open-cluster-management.io/v1
 kind: Subscription
 metadata:
   annotations:
-    apps.open-cluster-management.io/github-branch: master
+    #apps.open-cluster-management.io/github-branch: master
+    apps.open-cluster-management.io/github-branch: io-pattern-busybox
```
[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/app-busybox-1/subscriptions/busybox/subscription.yaml:7]: sorry, leftovers from testing, I need to revert it

File: rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml [hunk 3]
```diff
@@ -6,9 +6,11 @@ metadata:
   labels:
     app: busybox-sample
 spec:
-  preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  #preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  preferredCluster: "sagrawal-c1"
```
[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml:10]: leftover from testing. I will clean it in final push.

File: rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml (hunk 1 above)
Comment (shylesh) [reply rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml:13]: leftover from testing. I will clean it in final push.

[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml:13]: leftover from testing. I will clean it in final push.

File: rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml:10]: leftover from testing. I will clean it in final push.

File: rdr/busybox/scripts/simple_io.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    do
+        dd if=/dev/urandom of=$file bs=$(($RANDOM)) count=$(($RANDOM%3+1))
+        md5sum $file >>$hashfile
+        sync
+        sleep 15
+        if [ $(($RANDOM%2)) == 1 ]
+        then
+            md5sum -c $hashfile
+        fi
+    done
```
[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/scripts/simple_io.sh:13]: for now, pod logs shows the output of those commands. However, from ocs-ci side I will be implementing the checksum verification then defenitely it should take care of printing the necessary info.

File: rdr/busybox/scripts/simple_io.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+while true
+    file=/mnt/test/data_`date +%s`
+    hashfile=/mnt/test/hashfile
+    do
+        dd if=/dev/urandom of=$file bs=$(($RANDOM)) count=$(($RANDOM%3+1))
+        md5sum $file >>$hashfile
+        sync
+        sleep 15
```
[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/scripts/simple_io.sh:8]: ack, I will do that.

[the remark this answers is not in the document]
Comment (shylesh) [reply rdr/busybox/scripts/simple_io.sh]: done

File: rdr/busybox/scripts/simple_io.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    file=/mnt/test/data_`date +%s`
+    hashfile=/mnt/test/hashfile
+    do
+        dd if=/dev/urandom of=$file bs=$(($RANDOM)) count=$(($RANDOM%3+1))
+        md5sum $file >>$hashfile
+        sync
+        sleep $(($RANDOM%15+1))
+        if [ $(($RANDOM%2)) == 1 ]
+        then
+            md5sum -c $hashfile
```
Comment (prsurve) [inline rdr/busybox/scripts/simple_io.sh:11]: are we planning to add --quite

**Review by sidhant-agrawal — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

File: rdr/busybox/scripts/simple_io.sh (hunk 6 above)
Comment (shylesh) [reply rdr/busybox/scripts/simple_io.sh]: done

**Pull request merged**